api/s2n.h

```diff
@@ -333,6 +333,39 @@ S2N_API
 extern int s2n_connection_set_read_fd(struct s2n_connection *conn, int readfd);
 S2N_API
 extern int s2n_connection_set_write_fd(struct s2n_connection *conn, int writefd);
+
+/**
+  * Gets the assigned fd for the read channel of an s2n connection.
```

Comment:
Should we write out "file descriptor" at least once in this documentation?

Reply:
done